### PR TITLE
dialog and styles refactoring

### PR DESCRIPTION
**Pull request by keblodev**

### CODE REVIEW

File: _client/client_app/angular_app/elements/iwish-core/iwish-dialogs/iwish-dialog/iwish-dialog.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        var selectedCategoryObj = categories.filter(
+            function(item, index){
+              item.index = index;
+              return item.selected;
+            })[0];
+        this.selectedCategory = selectedCategoryObj.index;
+      },
+
+			showDialogChanged: function(oldVal, newVal) {
+        console.log('changed');
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-core/iwish-dialogs/iwish-dialog/iwish-dialog.html]: remove please console

File: _client/client_app/angular_app/elements/iwish-core/iwish-dialogs/iwish-dialog/iwish-dialog.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        </div>
+
+        <core-media-query query="min-width: {{responsiveWidth}}" queryMatches="{{isWide}}"></core-media-query>
+	</template>
+	<script>
+
+		Polymer('iwish-dialog', {
+      responsiveWidth:    '50em',
+
+      heading: 'looool',
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-core/iwish-dialogs/iwish-dialog/iwish-dialog.html]: remove heading if you dont use it

**Comment by alex-sirotenko** [comment]
+1

File: _client/client_app/angular_app/elements/core.html [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
 <link rel="import" href="../bower_components/polymer/polymer.html">
 <link rel="import" href="../bower_components/core-media-query/core-media-query.html">
-<link rel="import" href="../bower_components/core-item/core-item.html">
 <link rel="import" href="../bower_components/core-list/core-list.html">
 <link rel="import" href="../bower_components/core-scroll-threshold/core-scroll-threshold.html">
+
+<link rel="import" href="../bower_components/core-item/core-item.html">
 <link rel="import" href="../bower_components/core-iconset-svg/core-iconset-svg.html">
 <link rel="import" href="../bower_components/core-icon/core-icon.html">
+<link rel="import" href="../bower_components/core-icon-button/core-icon-button.html">
```
Comment (ronanamsterdam) [inline _client/client_app/angular_app/elements/core.html]: testes

File: _client/client_app/angular_app/elements/iwish-core/iwish-dialogs/iwish-dialog/iwish-dialog.html (hunk 1 above)
Comment (ronanamsterdam) [inline _client/client_app/angular_app/elements/iwish-core/iwish-dialogs/iwish-dialog/iwish-dialog.html]: yfuy

**Pull request merged**